src/components/auth/register/Step.jsx

```diff
@@ -1,6 +1,7 @@
 import React from "react";
 import InputField from "../InputField";
```

Comment:
Can you please rename InputField to TextInputField? Because ImageField is also an input field, so we need to make a difference between them.

Also please rename all input fields, to be more precise like this:
InputField => TextInputField
ImageField => ImageInputField
CheckboxField => CheckboxInputField

Reply:
Please update the imports as well. 
Screenshot of the error:

Reply:
Yea I forgot to save the step.jsx